app/src/main/java/com/google/android/samples/socialite/repository/ChatRepository.kt

```diff
@@ -107,6 +108,12 @@ class ChatRepository @Inject internal constructor(
         )
 
         coroutineScope.launch {
+            // Special incoming message indicating to add shorts videos to try preload in exoplayer
+            if (text == "preload") {
```

Comment:
can we add an extra check here to see if the app is currently connected to the internet? maybe send a different message if text is preload, and app is not connected to the internet?

the experience is kind of weird right now if you send a 'preload' message and the app is not connected to the internet. on the home screen there ends up being a lot of blank videos.

Reply:
Added a FR (https://github.com/android/socialite/issues/92)

Reply:
Maybe we can track this separately (https://github.com/android/socialite/issues/92) to gracefully handle all non internet scenarios. Since "preload" is a specific user driven action, the user would be aware if the internet is not available. I would like this PR to be focussed on preload, since that is an independent feature devs need to look at.